source/options.tsx

```diff
@@ -238,8 +240,8 @@ async function generateDom(): Promise<void> {
 		select('#logHTTP-line')!.hidden = false;
 	}
 
-	// Add feature count. CSS-only features are added approximately
-	select('.features-header')!.append(` (${featuresMeta.length + 25})`);
+	// Add feature count
+	select('.features-header')!.append(` (${featuresMeta.length})`);
```

Comment:
We have a more precise count now that CSS features are also parsed from the readme.

Maybe a `+ 10` or something is still warranted to account for the small features in `refined-github.css`?

Reply:
plz no https://github.com/refined-github/refined-github/issues/4951#issuecomment-953289103